src/external-repos.md

```diff
@@ -29,6 +28,20 @@ to these tools should be filed against the tools directly in their respective
 upstream repositories. The exception is that when rustc changes are required to
 implement a new tool feature or test, that should happen in one collective rustc PR.
 
+`subtree` dependencies are currently managed by two distinct approaches:
+
+* Using `git subtree` directly
```

Comment:
```suggestion
* Using `git subtree`
```
There's no "indirect" usage so I find the additional qualifier confusing

Reply:
I wanted to distinguish "using subtrees via josh" and "using subtrees via git subtree", but the word was extraneous, I suppose. Removed it.